src/components/JournalPage/index.jsx

```diff
@@ -34,8 +35,8 @@ class JournalPage extends React.Component {
                     // rich text and raw HTML use the same component
                     return <RawHTML content={el.value} />;
                   case PDF:
-                    // return <PDF />
-                    return <div>PDF</div>;
+                    // TODO: need to change id for search
+                    return <PDFViewer id={el.id} url={el.value.url} title={el.value.block_title} />;
```

Comment:
made some changes in backend, so now every component has a span_id and a title so you can use these consistently between components. 
`<PDFViewer id={el.span_id} url={el.value.url} title={el.value.title} />;